src/less/components/breadcrumb/index.less

```diff
@@ -1,17 +1,22 @@
 /* ========== hxBreadcrumbs ========== */
 .hxBreadcrumb {
+  display: flex;
+  margin-top: 0.5rem;
+  margin-bottom: 1rem;
```

Comment:
Please hold off on applying any margin to component containers.

Margin is _external_ spacing _between_ components and relies on DOM nesting context in order for white space to be applied correctly.

- please remove `margin-top`
- please remove `margin-bottom`

Reply:
Removed margin and implemented spacing using padding.